### ISSUE TITLE
Output VACCINE table no results on vx_type

### ISSUE THREAD

**Issue by mmmartinp** [UMC-Utrecht-RWE/ConcePTION-Level3#13]
Hi @t-andresvaz @relbersumcu @acidroyo-UMCU,

We have noticed in all the html generated by QC L3 on VACCINES table in BIFAP that no results stratified by vx_type have been generated, only based on ATC_code.

For COVID19 vaccines we have the ATCcode, however, the rest of the vaccines are identified by their "vx_type" (as per the specifications of the CDM at least one of them, either ATC code or vxtype, have to be present in Vaccines table of teh CDM).

It could be useful to have results based not only on the ATCcode of vaccines but on the vx_type.

Thank you,

Mar

@anallg

**Comment by t-andresvaz** [comment]
> For COVID19 vaccines we have the ATCcode, however, the rest of the vaccines are identified by their "vx_type" (as per the specifications of the CDM at least one of them, either ATC code or vxtype, have to be present in Vaccines table of teh CDM).

Yes, the important thing right now is that we can see Cleary if all covid vaccines are there. We will came back to this in a latter moment.
